hadoop-hdds/server-scm/src/main/java/org/apache/hadoop/hdds/scm/ha/SCMRatisServerImpl.java

```diff
@@ -189,8 +199,11 @@ public RaftPeerId getPeerId() {
       return selfPeerId;
     }
 
+    /**
+     * This will be used when the SCM HA config are not defined.
+     */
     SCMHAGroupBuilder(final SCMHAConfiguration haConf,
-                      final ConfigurationSource conf) throws IOException {
+        final ConfigurationSource conf, String clusterId) throws IOException {
```

Comment:
Can you clarify why use `clusterId` as RaftGroup id?  Is it useful somehow later?

Reply:
The idea is to use any id defined by user config like service id. ClusterId will be generated by only one SCM in the cluster and propagated to whole system.